ietf/templates/doc/shepherd_writeup.txt

```diff
@@ -63,30 +65,30 @@ to answer all of them.
    to be handed off to the responsible Area Director?
 
 10. Several IETF Areas have assembled [lists of common issues that their
-    reviewers encounter][6]. Do any such issues remain that would merit specific
-    attention from subsequent reviews?
+    reviewers encounter][6]. Are any such issues still present in the document?
```

Comment:
This implies that the shepherd needs to evaluate the document for ALL mentioned issues and report back whether they have been resolved. That is a huge burden that very few people (if any) would be qualified to take on. Suggestion:

```suggestion
    reviewers encounter][6]. Please identify the topic(s) listed that are relevant to the document, to aid selection of reviewers.
```

Reply:
See rewording proposal above.